### PR TITLE
lrzip 0.641

### PR DESCRIPTION
**Pull request by SMillerDev**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by dtrodrigues** [comment]
http://ck.kolivas.org/apps/lrzip/lrzip-0.640/liblrzip.c
```
/*
   Copyright (C) 2012-2016,2018 Con Kolivas

   This program is free software; you can redistribute it and/or modify
   it under the terms of the GNU General Public License as published by
   the Free Software Foundation; either version 2 of the License, or
   (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License
   along with this program. If not, see <http://www.gnu.org/licenses/>.
*/
```

**Comment by SMillerDev** [comment]
Anyone with an idea why this could be failing?
```console
nasm -I../ASM/x86/ -Dx64 -g -f elf64 -o 7zCrcOpt_asm.o /private/tmp/lrzip-20210307-2953-4qw1fu/lrzip-0.641/lzma/ASM/x86/7zCrcOpt_asm.asm
mkdir -p .libs
cp 7zCrcOpt_asm.o .libs/
  CCLD     liblzma.la
./7zCrcOpt_asm.lo: line 1: -e: command not found
Making all in ASM/x86
  CCLD     liblzmaasm.la
ar: no archive members specified
usage:  ar -d [-TLsv] archive file ...
```

**Review by Bo98 — COMMENTED** [review]
This should fix the error:

```ruby
system "make", "SHELL=bash"
```

---

ARM:

```rb
args = %W[
  --disable-dependency-tracking
  --prefix=#{prefix}
]
args << "--disable-asm" unless Hardware::CPU.intel?
system "./configure", *args
```

File: Formula/lrzip.rb [hunk 1]
```diff
@@ -22,7 +22,9 @@ class Lrzip < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "lz4"
   depends_on "lzo"
+  depends_on "nasm"
```
Comment (Bo98) [inline Formula/lrzip.rb:27]:
```suggestion
  depends_on "nasm" => :build if Hardware::CPU.intel?
```

**Comment by SMillerDev** [comment]
More ideas @Bo98 ?

**Comment by Bo98** [comment]
Ok we fixed some of the problems, but it looks like it is now making an empty archive (liblzmaasm) - which Apple's version of `ar` does not support doing. This requires patching the Makefile to fix, and upstream warrants an upstream bug report.

**Comment by demhademha** [comment]
I believe that https://github.com/ckolivas/lrzip/commit/ef8773132edd5fef083dbbede8af5decd0d8e9e7 is the cause of this issue.

**Pull request closed without merging**